allennlp/data/vocabulary.py

```diff
@@ -210,20 +210,18 @@ def add_token_to_namespace(self, token: str, namespace: str='tokens') -> int:
             self._token_to_index[namespace][token] = index
             self._index_to_token[namespace][index] = token
             return index
-        else:
-            return self._token_to_index[namespace][token]
+        return self._token_to_index[namespace][token]
```

Comment:
You should disable it in `.pylintrc`, not inline, by the way, in case you didn't know you could do that.